oi/keymap.py

```diff
@@ -5,11 +5,17 @@
     DefaultButton,
 )
 
+controllerDRIVER = XBoxController
+controllerOPERATOR = XBoxController
 
 class Controllers:
-    DRIVER: int
-    OPERATOR: int
+    DRIVER = 0
```

Comment:
While it's alright to remove these 'types' if it's your personal preference, I would suggest typing variables with more complicated objects (like SwerveNode), since it leads to better readability and IDE completion.